demos/theme/theme-shrine.scss

```diff
@@ -16,8 +16,9 @@
 
 @import "../../packages/mdc-theme/color-palette";
 
-// TODO: Use theme mixin(s) instead
-$mdc-theme-primary: #fff;
-$mdc-theme-secondary: $material-color-orange-500;
+$mdc-theme-primary: #FCB8AB;
```

Comment:
Nit: uppercase hex to lowercase hex wherever these are used, to match other styles